src/analyze-action.ts

```
@@ -106,6 +108,42 @@ async function run() {
         config,
         logger
       );
+
+      if (config.debugMode) {
```

Is there ever a danger of running with `debugMode === true` on the runner? Presumably, if so, the runner would fail on the upload stage, but it would be nice if we could detect this and fail faster.

---

Note, this will be caught (if it is indeed a problem) by updating the `unguarded-action-lib.ql` query.

---

It should never be possible to be in debug mode when using the runner. In particular, note that the call to `initConfig` in `runner.ts` has gained a hard-coded value of `false` for the parameter specifying whether we are in debug mode.